metadata-ingestion/tests/unit/test_source.py

```diff
@@ -39,3 +39,46 @@ def get_report(self) -> SourceReport:
 
     def close(self) -> None:
         return super().close()
+
+
+def test_discretize_dict_values():
+    """Test the _discretize_dict_values static method."""
+    # Test data with various count values
+    test_dict = {
+        "dataset": {
+            "schemaMetadata": 5,
+            "status": 12,
+            "ownership": 3,
+        },
+        "chart": {
+            "status": 8,
+            "ownership": 1,
+        },
+    }
+
+    # Call the static method
+    result = SourceReport._discretize_dict_values(test_dict)
```

Comment:
this test looks super verbose

why not just do `assert result == {"dataset": ...`